src/Illuminate/Pagination/Paginator.php

```diff
@@ -140,37 +120,8 @@ public function hasMorePages()
      */
     public function toArray()
     {
-        return [
-            'current_page' => $this->currentPage(),
-            'data' => $this->items->toArray(),
-            'first_page_url' => $this->url(1),
-            'from' => $this->firstItem(),
+        return array_merge(parent::toArray(), [
             'next_page_url' => $this->nextPageUrl(),
```

Comment:
Here it's enough just to add the custom data related to `Paginator` class, the common ones come from the parent.